roles/spark-standalone-install/tasks/main.yml

```diff
@@ -0,0 +1,137 @@
+---
+  - name: spark | user | ssh-key | probe
+    become_user: "{{ spark_user }}"
+    shell: cat ~/.ssh/id_rsa.pub
+    register: ssh_key_probe
+
+  - name: spark | user | ssh-key | record
+    set_fact:
+      ssh_key: "{{ ssh_key_probe.stdout }}"
+
+  - name: spark | user | key exchange
+    become_user: "{{ spark_user }}"
+    authorized_key:
+      user: "{{ spark_user }}"
+      key: "{{ hostvars[item]['ssh_key'] }}"
+    with_items: "{{ play_hosts }}"
```

Comment:
ssh-key-exchange requires you to pass in a [specific group](https://github.com/Kitware/gobig/blob/master/roles/ssh-key-exchange/tasks/main.yml#L24 ).  Since that will always be the hosts selected for this role  it seemed redundant to require people creating playbooks to specify the hosts twice.  Maybe there is a way to default the group somehow and then include the role?